src/renderer/component/selectChannel/view.jsx

```diff
@@ -159,7 +159,6 @@ class ChannelSection extends React.PureComponent<Props, State> {
           <FormField
             key="channel"
             type="select"
-            tabIndex="1"
```

Comment:
What is this change fixing?

Reply:
It wasn't doing anything so I removed it.